modules/openxr/extensions/openxr_hand_tracking_extension.cpp

```diff
@@ -57,9 +58,11 @@ OpenXRHandTrackingExtension::~OpenXRHandTrackingExtension() {
 HashMap<String, bool *> OpenXRHandTrackingExtension::get_requested_extensions() {
 	HashMap<String, bool *> request_extensions;
 
-	request_extensions[XR_EXT_HAND_TRACKING_EXTENSION_NAME] = &hand_tracking_ext;
-	request_extensions[XR_EXT_HAND_JOINTS_MOTION_RANGE_EXTENSION_NAME] = &hand_motion_range_ext;
-	request_extensions[XR_FB_HAND_TRACKING_AIM_EXTENSION_NAME] = &hand_tracking_aim_state_ext;
+	bool enabled = GLOBAL_GET("xr/openxr/extensions/hand_tracking");
```

Comment:
The approach I've used in https://github.com/godotengine/godot/pull/82614 is to gate at creation time instead, so we don't instantiate an extension we never use.

See https://github.com/godotengine/godot/blob/f5696c311cdb09e0a34fa4ba7ef5d2524c515b89/modules/openxr/register_types.cpp#L108

Reply:
Yeah I guess that makes sense, it does mean we have to ensure to do a nullptr check every time we access the extension but I think we do that anyway